demos/common/export_models/export_model.py

```diff
@@ -222,6 +223,15 @@ def set_rt_info(model_folder_path, model_filename, config_filename):
     shutil.move(os.path.join(model_folder_path, temp_model_name), os.path.join(model_folder_path, model_filename))
     shutil.move(os.path.join(model_folder_path, temp_model_name.replace('.xml','.bin')), os.path.join(model_folder_path, model_filename.replace('.xml','.bin')))
 
+def get_models_max_context(tmpdirname, config_filename):
+    with open(os.path.join(tmpdirname, config_filename), 'r') as config_file:
+        config_data = json.load(config_file)
+        if config_data['max_position_embeddings'] is not None:
+            return config_data['max_position_embeddings']
+        if config_data['n_positions'] is not None:
+            return config_data['n_positions']
```

Comment:
Are these values mutually exclusive? Is `max_position_embeddings` more important if they aren't?

Reply:
they are exclusive. max_position_embeddings is more commonly used